hack/runtime-migrator/README-rb.md

```diff
@@ -106,27 +107,31 @@ The `backup-results.json` file contains the following content:
 ]
 
 ```
-The above example The runtime with the `exxe4b14-7bc2-4947-931c-f8673793b02f` identifier was not found ; the identifier may be incorrect, or the corresponding shoot was deleted for some reason.
+In the above example the runtime with the `exxe4b14-7bc2-4947-931c-f8673793b02f` identifier was not found ; it may be incorrect, or the corresponding shoot was deleted for some reason. 
```

Comment:
or the cluster was created by KIM

Reply:
I modified this sentence a bit to mention `kcp.provisioner.kyma-project.io/runtime-id` label